fastlane/swift/Fastlane.swift

```diff
@@ -4061,7 +4065,7 @@ func xcov(workspace: String? = nil,
           coverallsServiceJobId: String? = nil,
           coverallsRepoToken: String? = nil,
           xcconfig: String? = nil,
-          ideFoundationPath: String = "/Applications/Xcode-9.0.1.app/Contents/Developer/../Frameworks/IDEFoundation.framework/Versions/A/IDEFoundation",
+          ideFoundationPath: String = "/Applications/Xcode-10.1.app/Contents/Developer/../Frameworks/IDEFoundation.framework/Versions/A/IDEFoundation",
```

Comment:
Huh, what caused this change?

Reply:
Ah, this just happens based on the bumpers xcode version selected. It doesn't really matter at all since this gets regenerated on the client side when they use it 😊